lib/Minz/Request.php

```diff
@@ -71,6 +71,15 @@ public static function currentRequest() {
 			'params' => self::$params,
 		);
 	}
+	public static function modifiedCurrentRequest(array $extraParams = null) {
```

Comment:
Good idea; I had to double-check which type-hinting was available from which PHP version (this info is apparently gone from the current PHP documentation):
http://web.archive.org/web/20200101052646/https://www.php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration.types
We should remember to use it more, and then even more when we drop PHP 5.x.

Reply:
Ah, moved there :-) https://php.net/language.types.declarations

Reply:
About dropping PHP 5: https://github.com/FreshRSS/FreshRSS/discussions/3321